packages/ui5-tooling-transpile/lib/task.js

```diff
@@ -74,12 +74,20 @@ module.exports = async function ({ log, workspace /*, dependencies*/, taskUtil,
 
 	// replace the version in all files handled by this task because this plugin handles additional file types
 	// which are not supported by the replaceVersion task of the UI5 Tooling (hardcoded some selected file types)
-	// (HINT: do this a bit loosly coupled for now to avoid tight dependencies to UI5 Tooling)
+	// (HINT: do this a bit loosely coupled for now to avoid tight dependencies to UI5 Tooling)
+	// Also check for @ui5/builder under @ui5/cli to avoid issues with the module resolution
 	try {
 		// dynamically require the replaceVersion task
 		// (using the absolute path to the module to avoid issues with the module resolution)
-		const replaceVersion = (await import(pathToFileURL(require.resolve("@ui5/builder/tasks/replaceVersion"))))
-			.default;
+		const replaceVersion = (
+			await import(
+				pathToFileURL(
+					require.resolve("@ui5/builder/tasks/replaceVersion", {
+						paths: [cwd, `node_modules${path.sep}@ui5${path.sep}cli`]
```

Comment:
hmm, this hard-coded path is not optimal - the CLI isn't necessarily part of the `node_modules` of the project - it can also be installed globally. I think we need to resolve the ' @ui5/cli` beforehand.

Reply:
Please do it like that: 

```js
		const replaceVersion = (
			await import(
				pathToFileURL(
					require.resolve("@ui5/builder/tasks/replaceVersion", {
						paths: [cwd, path.dirname(require.resolve("@ui5/cli/package.json"))]
					})
				)
			)
		).default;
```

this ensures that the `@ui5/cli` path is resolved either locally or globally, depending how it has been installed.

Reply:
Thanks @petermuessig. Updated as suggested.